classes/Infrastructure/Auth/MockableAuthenticator.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2017 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Infrastructure\Auth;
+
+use OpenCFP\Domain\Services\Authentication;
+use OpenCFP\Domain\Services\NotAuthenticatedException;
+
+class MockableAuthenticator implements Authentication
```

Comment:
I really like this, but maybe it should be moved to the ```OpenCfp\Test\Helper``` namespace?

Reply:
👍 to that suggestion

Reply:
All right, I will move it.